### PR TITLE
desktop page portfolio

### PR DESCRIPTION
**Pull request by frnega**
tried to do desktop page to look like the design

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,480 +1,1243 @@
-* {
-  margin: 0;
-  padding: 0;
-}
+@media only screen and (min-width: 760px) {
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] As requested in the requirements, please adjust your media query to 768px for the best responsiveness on all mobile devices. 😀
az

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @frnega    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frnega** [comment]
changed the media size to 768

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @frnega     , 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**